base/docs/helpdb/Base.jl

```diff
@@ -5643,7 +5643,8 @@ julia> deleteat!([6, 5, 4, 3, 2, 1], 1:2:5)
 
 julia> deleteat!([6, 5, 4, 3, 2, 1], (2, 2))
 ERROR: ArgumentError: indices must be unique and sorted
- in deleteat! at array.jl:543
+ in deleteat!(::Array{Int64,1}, ::Tuple{Int64,Int64}) at ./array.jl:571
+ in eval(::Module, ::Any) at ./boot.jl:231...
```

Comment:
Before this PR I'm seeing a handful, but quite a bit fewer than this PR is adding:
```
$ git grep -n 'in eval'
base/test.jl:914: in eval(::Module, ::Any) at ./boot.jl:226
doc/manual/mathematical-operations.rst:388:     in eval at ./boot.jl:263
doc/manual/mathematical-operations.rst:396:     in eval at ./boot.jl:263
doc/manual/mathematical-operations.rst:401:     in eval at ./boot.jl:263
doc/manual/mathematical-operations.rst:416:     in eval at ./boot.jl:263
doc/stdlib/test.rst:283:        in eval(::Module, ::Any) at ./boot.jl:226
test/core.jl:336:@test_throws ErrorException eval(:(abstract Qux2_{T} <: Sup2b_{Qux2_{Int}, T})) # wrapped in eval to avoid #16793
```